### PR TITLE
Lesson one branch

### PR DESCRIPTION
**Pull request by arolik**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (StanislawVictorovich) [inline index.html:1]: index.html not needed. Please remove it out

File: scripts/LessonOnePartOne.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (StanislawVictorovich) [inline scripts/LessonOnePartOne.js:29]: remove unnecessary line breaks

File: scripts/LessonOnePartTwo.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*Написать функцию, которая принимает массив числовых значений и возвращает среднее значение */ 
+
+let arr = [2,4,3,5,6,8,7,9];
+
+function findAverage(arr){
+    let num = 0;
+    for(let i=0; i<arr.length; i++){
+        num += arr[i];
+    }
+    let result = num / arr.length;
```
Comment (StanislawVictorovich) [inline scripts/LessonOnePartTwo.js:10]: remove result variable and make direct return

File: scripts/LessonOnePartTwo.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (StanislawVictorovich) [inline scripts/LessonOnePartTwo.js:30]: remove unused line breaks

File: scripts/LessonOnePartTwo.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+/*Написать функцию, которая принимает массив числовых значений и возвращает среднее значение */ 
+
+let arr = [2,4,3,5,6,8,7,9];
+
+function findAverage(arr){
+    let num = 0;
+    for(let i=0; i<arr.length; i++){
```
Comment (StanislawVictorovich) [inline scripts/LessonOnePartTwo.js:7]: please add whitespaces around the operators

File: index.html (hunk 1 above)
Comment (arolik) [reply index.html:1]: deleted first line. if i right undestand it )

File: scripts/LessonOnePartOne.js (hunk 2 above)
Comment (arolik) [reply scripts/LessonOnePartOne.js:29]: remove

File: scripts/LessonOnePartTwo.js (hunk 3 above)
Comment (arolik) [reply scripts/LessonOnePartTwo.js:10]: wrote result in direct return

File: scripts/LessonOnePartTwo.js (hunk 4 above)
Comment (arolik) [reply scripts/LessonOnePartTwo.js:30]: remove

File: scripts/LessonOnePartTwo.js (hunk 5 above)
Comment (arolik) [reply scripts/LessonOnePartTwo.js:7]: add whitespace

**Pull request merged**